pom.xml

```diff
@@ -84,6 +84,9 @@
     </modules>
 
     <properties>
+        <sonar.organization>acrolinx</sonar.organization>
+        <sonar.projectKey>acrolinx-flexmark-java</sonar.projectKey>
+        <sonar.host.url>https://sonarcloud.io</sonar.host.url>
```

Comment:
According to https://docs.sonarsource.com/sonarqube/latest/analyzing-source-code/analysis-parameters/#server-connection this should be the default value, so we can omit it?